src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -1609,43 +1612,18 @@ static bool addBsonFilter
   }
   else if (opr == OPR_EXIST)
   {
-    if (std::string(right) == ENT_ENTITY_TYPE)
-    {
-      // Special case: entity type
```

Comment:
Not really part of this PR, but a bit of cleanup regarding `+type` removal.

NTC